### PR TITLE
Add blockchain name to filters, change color to white, remove useless…

### PR DESCRIPTION
**Pull request by Milena-Wasiak**
Add blockchain name to filters, change color to white, remove useless code, remove blockchain under market value

### CODE REVIEW

**Review by agatascalac — APPROVED** [review]
👍

File: src/components/ChartDescription/ChartDescription-view.tsx [hunk 1]
```diff
@@ -12,7 +12,6 @@ const ChartDescriptionView = ({ type, blockchain }: ViewProps) => {
       <Grid container spacing={1} alignItems="center" className={classes.root}>
         <Grid item xs={1}><div className={classesDot}/></Grid>
         <Grid item xs={11} className={classes.type}>{`${type} volume`}</Grid>
-        <Grid item xs={12} className={classes.block}>{blockchain}</Grid>
```
Comment (dariuszsobkowicz) [inline src/components/ChartDescription/ChartDescription-view.tsx:15]: if `blockchain` is not in use please remove from props ;)

File: src/components/DetailsMenu/DetailsMenu-view.tsx [hunk 2]
```diff
@@ -44,10 +43,13 @@ const DetailsMenu = ({ id, address, description, updateDescription, type, blockc
       <Grid item xs={1}>
         <ChartDescription type={type} blockchain={blockchain} />
       </Grid>
-      <Grid container direction="row" justify="flex-start" alignItems="center" className={classesFilters}>
+      <Grid container direction="row" justify="flex-start" alignItems="center" className={classesInfo}>
         <Grid item xs={1}>
```
Comment (dariuszsobkowicz) [inline src/components/DetailsMenu/DetailsMenu-view.tsx:47]: is white color better then grey?

File: src/components/DetailsMenu/DetailsMenu-view.tsx [hunk 3]
```diff
@@ -44,10 +43,13 @@ const DetailsMenu = ({ id, address, description, updateDescription, type, blockc
...
           FILTERS
         </Grid>
+        <Grid item xs={1}>
+          {blockchain}
+        </Grid>
```
Comment (dariuszsobkowicz) [inline src/components/DetailsMenu/DetailsMenu-view.tsx:52]: blockchain name should start with uppercase or lowercase?

**Review by dariuszsobkowicz — CHANGES_REQUESTED** [review]

File: src/components/DetailsMenu/DetailsMenu-view.tsx (hunk 2 above)
Comment (Milena-Wasiak) [reply src/components/DetailsMenu/DetailsMenu-view.tsx:47]: yes, like Mat said on demo

**Review by wojciech-nowakowski — APPROVED** [review]

**Review by dariuszsobkowicz — APPROVED** [review]
well done

**Pull request merged**